<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettingsDialog</class>
 <widget class="QDialog" name="UISettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chkAutoOpenRecentOnStart">
         <property name="text">
          <string>Automatically open most recent profile when GUI starts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMinimizeToTrayOnStart">
         <property name="text">
          <string>Minimize application to tray when GUI starts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMinimizeToTray">
         <property name="text">
          <string>Minimize application to tray</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMinimizeToTrayOnClose">
         <property name="text">
          <string>Minimize application to tray when close button pressed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAlwaysShowTray">
         <property name="text">
          <string>Always show tray icon</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_SSL">
      <attribute name="title">
       <string>SSL Certificate&amp;&amp;Key</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>You can modify/generate your personal default SSL private key and certificate below:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="grpSSL" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnBrowsePrivateKeyFile">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="edtPrivateKeyFile">
            <property name="maxLength">
             <number>1024</number>
            </property>
            <property name="placeholderText">
             <string>RSA key in PEM format, no passphrase</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>SSL certificate file:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Private key file:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnBrowseSSLCertFile">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <layout class="QHBoxLayout" name="hLayout_GenerateKeyCert">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="hSpacer_GenerateKeyCert">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnGenerateKeyCert">
              <property name="text">
               <string>Generate new private key &amp;&amp; certificate pair...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edtSSLCertFile">
            <property name="maxLength">
             <number>1024</number>
            </property>
            <property name="placeholderText">
             <string>in PEM format</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>label_7</zorder>
         <zorder>edtSSLCertFile</zorder>
         <zorder>btnBrowseSSLCertFile</zorder>
         <zorder>label_8</zorder>
         <zorder>edtPrivateKeyFile</zorder>
         <zorder>btnBrowsePrivateKeyFile</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UISettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UISettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
